card/mod/01_core/set/all/tracked_attributes.rb

```diff
@@ -27,20 +40,38 @@ def assign_set_specific_attributes
   end
 end
 
-def extract_subcard_args! args={}
-  extracted_subcards = args.delete('subcards') || {}
+protected
+
+def extract_subcard_args! args
+  subcards = args.delete('subcards') || {}
   args.keys.each do |key|
     if key =~ /^\+/
-      val = args.delete key
-      val = { 'content' => val } if String === val
-      extracted_subcards[key] = val
+      subcards[key] = args.delete(key)
     end
   end
-  extracted_subcards
+  subcards
 end
 
+def extract_type_id! args = {}
```

Comment:
Assignment Branch Condition size for extract_type_id! is too high. [17.2/15]<br>Method has too many lines. [16/15]<br>Surrounding space detected in default value assignment.